package.json

```diff
@@ -4,7 +4,7 @@
     "email": "info@sussol.net"
   },
   "name": "mSupplyMobile",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Pretty sure our clever android build number logic doesn't like that, I've tried with Tupaia App before. Probably not hard to fix that though.